### DISCUSSION TITLE
Won't work on Lenovo P53s

### DISCUSSION THREAD

**Discussion by OSE-NZ** [danielewood/sierra-wireless-modems#34] [General]
I cannot get the EM7455 to be recognized on a Lenovo P53s.

I used an external enclosure and run your setup script which worked great.  The modem works fine in an external enclosure on the P53s.  So no problems with the EM7455 modem itself.

If I tape up the PCIe and USB 3.0 pins on the L850-GL modem (to force USB mode), the laptop halts during boot with an unauthorized network card installed.  This is expected because of the whitelisting issue.  If I boot the laptop without the modem first and then plug the modem in (with the pins taped), the modem shows up on the USB bus.  This proves to me that the USB pins on the M.2 port are functional.

However, if I try the same thing with the EM7455 nothing shows up on the USB bus.  It doesn't matter if I boot if without the modem first or with the modem.  No BIOS white list halt and no device on the USB bus.  I can't work out how this is possible.  I checked the data sheets and all the power and enable pins appear to be the same.  It's got me stumped.

Do you have any idea what could be stopping the EM7455 from enumerating on the USB bus?  There is nothing in dmesg or lsusb with the EM7455 but everything is normal with the L850-GL.

**Comment by aussetg** [comment]
I have the exact same problem on the T14s. Did you find any solution ?
